### PR TITLE
Show observation reference in table instead of program reference + observation id

### PR DESCRIPTION
**Pull request by rpiaggio**
image

### CODE REVIEW

File: modules/web/client-model/src/main/scala/observe/ui/model/ObsSummary.scala [hunk 1]
```diff
@@ -82,8 +82,7 @@ object ObsSummary:
       observingMode      <- c.get[Option[ObservingMode]]("observingMode")
       visualizationTime  <- c.get[Option[Timestamp]]("visualizationTime")
       posAngleConstraint <- c.get[PosAngleConstraint]("posAngleConstraint")
-      programReference   <-
-        c.downField("program").downField("reference").get[Option[ProgramReference]]("label")
+      obsReference       <- c.downField("reference").get[Option[ObservationReference]]("label")
```
Comment (cquiroz) [inline modules/web/client-model/src/main/scala/observe/ui/model/ObsSummary.scala:85]: These may fail if `reference` is null

Comment (rpiaggio) [reply modules/web/client-model/src/main/scala/observe/ui/model/ObsSummary.scala:85]: Doesn't it return `None`? Anyway, this is defensive. Ready observations are expected to always have a `reference`.

Comment (rpiaggio) [reply modules/web/client-model/src/main/scala/observe/ui/model/ObsSummary.scala:85]: I can confirm that it returns `None` instead of failing.

**Review by cquiroz — APPROVED** [review]
LGTM

**Pull request merged**